changes/573cc999aba312ea0a8e5c3766303447.yaml

```diff
@@ -0,0 +1,6 @@
+---
+desc: '- Updated ``$lib.axon.put()`` to check for the existence of the bytes in the
```

Comment:
What's this leading dash about?

Reply:
I don't understand your question? It's the "bullet" for the changelog entry.

Reply:
nm. too many ways of doing the same thing